tensorflow_datasets/image/ham10000.py

```diff
@@ -0,0 +1,112 @@
+"""HAM10000 Dataset"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import os
+import csv
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_URL = ("https://www.kaggle.com/kmader/skin-cancer-mnist-ham10000/"
+        "downloads/skin-cancer-mnist-ham10000.zip/2")
+
+_DESCRIPTION = ("The dataset consists of 10015 dermatoscopic images which"
+                "can serve as a training set for academic machine learning purposes."
+                "Cases include a representative collection of all important diagnostic "
+                "categories in the realm of pigmented lesions: Actinic keratoses and " 
+                "intraepithelial carcinoma / Bowen's disease (akiec), basal cell carcinoma (bcc), "
+                "benign keratosis-like lesions (solar lentigines / seborrheic "
+                "keratoses and lichen-planus like keratoses, bkl), dermatofibroma (df), "
+                "melanoma (mel), melanocytic nevi (nv) and vascular lesions "
+                "(angiomas, angiokeratomas, pyogenic granulomas and hemorrhage, vasc)")
+
+_IMAGE_SHAPE = (450, 600, 3)
+
+_NAMES = ['akiec', 'bcc', 'bkl', 'df', 'mel', 'nv', 'vasc'] 
+
+_CITATION = """\
```

Comment:
Hi! Thank you so much for going forward with this. I was just about to start working on implementing the HAM10000 dataset and found you already did the work. One request: Please change the citation to the manuscript of the workup of the data:

```suggestion
_CITATION = """\
    @article{Tschandl2018_HAM10000,
        author = {Philipp Tschandl and Cliff Rosendahl and Harald Kittler},
        title = {The {HAM10000} dataset, a large collection of multi-source 
            dermatoscopic images of common pigmented skin lesions},
        journal = {Sci. Data},
        volume = {5},
        year = {2018},
        pages = {180161},
        doi = {https://doi.org/10.1038/sdata.2018.161}
        }
"""
```

Reply:
@ptschandl  Thank you so much for appreciating and reviewing my contribution, I will make the suggested changes